examples/next-partial-prerendering/README.md

```diff
@@ -0,0 +1,23 @@
+## Next.js Partial Prerendering
+
+This is a demo of [Next.js](https://nextjs.org) using [Partial Prerendering](https://nextjs.org/docs/app/api-reference/next-config-js/partial-prerendering).
+
+This template uses the new Next.js [App Router](https://nextjs.org/docs/app). This includes support for enhanced layouts, colocation of components, tests, and styles, component-level data fetching, and more.
+
+It also uses the experimental Partial Prerendering feature available in Next.js 14. Partial Prerendering combines ultra-quick static edge delivery with fully dynamic capabilities and we believe it has the potential to [become the default rendering model for web applications](https://vercel.com/blog/partial-prerendering-with-next-js-creating-a-new-default-rendering-model), bringing together the best of static site generation and dynamic delivery.
+
+> ⚠️ Please note that PPR is an experimental technology that is not recommended for production. You may run into some DX issues, especially on larger code bases.
+
+## How it works
```

Comment:
Is it worth adding the "known issue" about not working with Wrangler local dev  here? Or is this literally just copied over from the original example?

Reply:
Just a copy.
We should probably have a KI section on opennext.js.org, it will be more visible there.
_(I created #309 for local dev.)_